simple_config

```diff
@@ -1,3 +1,3 @@
 server {
-    listen  80
-}
\ No newline at end of file
+    listen  8080;
```

Comment:
Our tests currently assume/hardcode this port number. We'll need to figure out a way to get rid of this dependency, so that we can test with any port number. That way, our tests won't be as brittle to changes in config files, etc.